templates/sysv/lsb-3.1/init.sh

```diff
@@ -88,7 +88,13 @@ stop() {
       sleep 1
     done
     if status ; then
-      emit "$name stop failed; still running."
+      if [ "$KILL_ON_STOP_TIMEOUT" -eq 1 ] ; then
+        trace "Timeout reached. Killing $name (pid $pid) with SIGKILL"
```

Comment:
I wonder about making the language stronger. "Killing $name (pid $pid) with SIGKILL. This may cause data loss in whatever $name is doing when killed."

Thoughts?